docs/guides/cloud/introduction.mdx

```diff
@@ -49,13 +49,15 @@ Check out the <Icon name="github" />
 ### Analyze and diagnose
 
 Store the full history of your test results, with
-[video clips, screenshots, and full stack traces](/guides/cloud/runs#Test-failures).
+[video clips, screenshots, and full stack traces](/guides/cloud/recorded-runs#Attempts-and-errors).
```

Comment:
Based on linked in the public site, would have expected this to be `guides/cloud/recorded-runs#Artifacts`

Reply:
Ely took a stab at updating the introduction page, here https://github.com/cypress-io/cypress-documentation/pull/5121. Can you cross reference these changes with that? This PR isn't considering content changes, just the IA.